packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/hooks/post_gen.dart

```diff
@@ -1,6 +1,23 @@
+import 'dart:io';
+
 import 'package:mason/mason.dart';
 
-void run(HookContext context) {
+void run(HookContext context) async {
   // Add your post-generation code here
   // (Code executed immediately after the generation step)
+
+  //Builds generated packages
+  if (context.vars['enable_feature_counter'] == false) {
```

Comment:
Please sync this with @RomanovaPrime

Reply:
It looks like that the post_gen is not working as expected so the code has been moved to the main command execution - https://github.com/Prime-Holding/rx_bloc/pull/431